### PR TITLE
Adding a yellow scale affect to the countdown below 5 seconds

### PR DESCRIPTION
**Pull request by jbruwer**

### CODE REVIEW

File: src/js/ui.ts [hunk 1]
```diff
@@ -460,8 +463,25 @@ export class Ui {
...
-    this.timerEl.textContent = value.toString();
+  updateTimer(value: number, updateCountDownTimer = false,
+        addFlashAnimation = false) {
+    if (addFlashAnimation) {
+      this.timerEl.textContent = value.toString();
+      this.timerFlashEl.textContent = value.toString();
+
+      const timerFlashAnimationEnded = (e: Event) => {
+        this.timerFlashEl.style.display = 'none';
+        removeClass(this.timerFlashEl,'flash');
```
Comment (Japegrape) [inline src/js/ui.ts]: Add whitespace.

File: src/js/ui.ts [hunk 2]
```diff
@@ -460,8 +463,25 @@ export class Ui {
...
+        this.timerFlashEl.removeEventListener('animationend',
+            timerFlashAnimationEnded);
+      };
+      this.timerFlashEl.style.display = 'block';
+      this.timerFlashEl.addEventListener('animationend',
+          timerFlashAnimationEnded);
+      addClass(this.timerFlashEl,'flash');
```
Comment (Japegrape) [inline src/js/ui.ts]: Add whitespace.

**Review by Japegrape — APPROVED** [review]

File: src/js/ui.ts (hunk 1 above)
Comment (jbruwer) [reply src/js/ui.ts]: Done

File: src/js/ui.ts (hunk 2 above)
Comment (jbruwer) [reply src/js/ui.ts]: Done

**Pull request merged**